docs/jade-getting-started.md

```diff
@@ -232,7 +235,7 @@ environment.  You'll actually spin up this instance on broad-jade-dev in next st
 
 > These instructions have not been tested yet! This may be a good step to
```

Comment:
Is it time to remove this disclaimer?

Reply:
They're still new - I don't think they've been tested.